src/classic/clvm_tools/cmds.rs

```diff
@@ -293,6 +445,12 @@ pub fn cldb(args: &[String]) {
             .set_type(Rc::new(PathOrCodeConv {}))
             .set_help("path to symbol file".to_string()),
     );
+    parser.add_argument(
+        vec!["-t".to_string(), "--tree".to_string()],
+        Argument::new()
+            .set_action(TArgOptionAction::StoreTrue)
+            .set_help("new style hierarchial view".to_string()),
```

Comment:
"new style hierarchial view" -- of what

mention that it's for running chialisp in the debugger.